terraform-sentinel/intro.md

```diff
@@ -0,0 +1,19 @@
+[Sentinel](https://www.hashicorp.com/sentinel/) is the Policy-as-Code product from HashiCorp that enforces automated enforcement of logic-based policy decisions across all HashiCorp products.
+
+It allows customers to implement policy-as-code in the same way that Terraform implements infrastructure-as-code.
+
+The Sentinel Command Line Interface (CLI) allows you to apply and test Sentinel policies including those that use mocks generated from Terraform Cloud and Terraform Enterprise plans.
+
+This **Sentinel & Terraform Cloud** environment provides you a place to create and test policies for your infrastructure in **30 minutes**.
+
+This scenario follows the **Team & Governance** track in the [Terraform Learn guide](https://learn.hashicorp.com/terraform/sentinel/sentinel-policies).
+
+Reference Materials:
+
+- [Sentinel documentation](https://docs.hashicorp.com/sentinel)
+
+You may want to try the following tutorials:
+
+- [Sentinel Policies](https://learn.hashicorp.com/vault/security/iam-sentinel)
+
+<img src="https://s3.amazonaws.com/terramino.hashicorp.fun/HashiCorp.png" alt="HashiCorp Logo"/>
```

Comment:
This URL is incorrect and gives a 404. Consider linking to a permanent, stable copy of the logo rather than directly to an S3 bucket. (Or skip it altogether.)